app/uk/gov/hmrc/pensionschemereturn/audit/PsrPostAuditEvent.scala

```diff
@@ -36,6 +37,7 @@ case class PsrPostAuditEvent(
     val apiDetails =
       Json.obj(
         "PensionSchemeTaxReference" -> pstr,
+        "SchemeName" -> schemeName,
```

Comment:
Ticket mentions more details, as:
```
"pensionSchemeAdministratorID" : "A2100005",
"schemeAdministratorName" : "ABC Hotels Ltd",
"pensionSchemeTaxReference" : "00000042IN",
"credentialRolePsaPsp" : "PSA",
"affinityGroup" : "Individual", // or Organisation
"schemeName" : "Open Scheme With AFT Already Submitted in TPSS",
"taxYear" : "2023-2024",
```
Do we need to include those too? Like here: https://github.com/hmrc/pension-scheme-return-frontend/blob/86947e8d1cf795567392a45b66502d7f308024f1/app/models/audit/PSRCompileAuditEvent.scala#L39